package.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "@vtex/cli-plugin-submit",
   "description": "Toolbelt plugin for the 'submit' command",
-  "version": "0.2.0",
+  "version": "0.3.0-beta.3",
```

Comment:
I think we should release a new major, because 2.x won't work with explicit dependencies AFAIK. WDYT @williamads @arturpimentel ?

Reply:
Could you explain why do you think it? There are plugins that use explicit dependencies. E.g autoupdate, edition, functions, infra... I think we will have incompatible version issues.